ProcessMaker/screen-builder
# FOUR-14287 Fix sort of pages in dropdown

## Reordering pages in the modal is not reflected in the menu list

## Solution
- Add a calculated property with the sorted pages
- Use the object reference to the page to select the page

## How to Test
1. Create multiple pages
2. Sort the pages
3. Use the drop-down of pages check it has the same order as the step 2

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-14287

## Code Review Checklist
